README.adoc

```diff
@@ -174,6 +174,29 @@ this example). The option definition would then look like this:
 flag :v, :verbose, 'be verbose (use up to three times)', multiple: true
 --------------------------------------------------------------------------------
 
+==== Skipping option parsing ====
+
+If you want to skip option parsing for your command or subcommand, you can add
+the `skip_option_parsing` method to your command definition and everything on your
+command line after the command name will be passed to your command as arguments.
+
+[source,ruby]
+-------------------------------------------------------------------------------
+command = Cri::Command.define do
+  name        'dostuff'
+  usage       'dostuff [args]'
+  aliases     :ds, :stuff
+  summary     'does stuff'
+  description 'This command does a lot of stuff, but not option parsing.'
+
+  skip_option_parsing
+
+  run do |opts, args, cmd|
+    puts args.inspect
+  end
+end
+-------------------------------------------------------------------------------
+
```

Comment:
I think this one could also benefit from an example, like:

> When invoking this command with `pet --animal=cat -f very-fuzzy`, the result options hash will be empty (`{}`), while the arguments array will be `["pet", "--animal=cat", "-f", "very-fuzzy"]`.

Reply:
Good idea, I've added a bit at the end explaining it